tox.ini

```diff
@@ -21,7 +21,7 @@ commands = pytest -v --tb native -s {posargs} {toxinidir}/tests/unit
 
 [testenv:integration]
 deps =
-    juju
+    python-libjuju
```

Comment:
```suggestion
    # Temporarily use branch until PR #507 is merged and released
    https://github.com/DomFleischmann/python-libjuju/archive/refs/heads/local-upgrades.zip#egg=juju
```

Reply:
The juju team has been really quick this time and already released those changes, so this won't be necessary.